components/item.tsx

```
@@ -24,31 +29,36 @@ export default function Item(props: ItemProps) {
+      <div className="absolute inset-0 bg-gradient-to-br from-gray-50/60 via-white/90 to-gray-100/80 dark:from-gray-900/60 dark:via-gray-800/80 dark:to-black/80 transition-all duration-700" />
+
+      <div
+        className="absolute inset-0 opacity-10 dark:opacity-20"
+        style={{
+          backgroundImage:
+            "url(\"data:image/svg+xml,%3Csvg width='40' height='40' viewBox='0 0 40 40' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='%23000000' fill-opacity='0.05' fill-rule='evenodd'%3E%3Cpath d='M0 0h40v40H0V0zm1 1h38v38H1V1z' /%3E%3C/g%3E%3C/svg%3E\")",
+        }}
+      />
+
+      <div className="absolute inset-0 bg-gradient-to-br from-blue-50/30 via-transparent to-purple-50/30 dark:from-blue-900/10 dark:via-transparent dark:to-purple-900/10 opacity-0 group-hover:opacity-100 transition-all duration-700 ease-out">
+        <div className="absolute -top-20 -right-20 w-40 h-40 rounded-full bg-blue-400/10 dark:bg-blue-400/5 blur-2xl group-hover:animate-pulse-slow opacity-0 group-hover:opacity-100 transition-opacity duration-1000 ease-in-out"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)